fastembed/rerank/cross_encoder/onnx_text_model.py

```diff
@@ -0,0 +1,58 @@
+from typing import Sequence, Dict, List
+
+import numpy as np
+
+from fastembed.common.onnx_model import OnnxModel, OnnxProvider
+from onnxruntime import InferenceSession
+from tokenizers import Tokenizer, Encoding
+from tokenizers.models import BPE
+from tokenizers.pre_tokenizers import Whitespace
+from tokenizers.processors import TemplateProcessing
+
+class OnnxCrossEncoderModel(OnnxModel):
+    def __init__(
+            self, 
+            model_name: str, 
+            cache_dir: str = None, 
+            threads: int = None, 
+            providers: Sequence[OnnxProvider] = None, 
+            **kwargs):
+        super().__init__()
+        self.tokenizer = self._load_tokenizer(model_name, cache_dir)
+        self.model = self._load_model(model_name, cache_dir, providers)
+
+    def _load_tokenizer(self, model_name: str, cache_dir: str = None) -> Tokenizer:
+        tokenizer = Tokenizer(BPE())
+        tokenizer.pre_tokenizer = Whitespace()
+        tokenizer.post_processor = TemplateProcessing(
+            single="[CLS] $A [SEP]",
+            pair="[CLS] $A [SEP] $B:1 [SEP]:1",
+            special_tokens=[
+                ("[CLS]", 1),
+                ("[SEP]", 2),
+            ],
+        )
+        return tokenizer
```

Comment:
we should already have tokenizer stored in `self.tokenizer`, we just need to configure it

so we won't need to initiate bpe, whitespace, etc, we just need to set up a post_processor

special_tokens are not just random numbers, but should be taken from a mapping, e.g. via `self.tokenizer.token_to_id("[CLS]"))`